lib/index.js

```diff
@@ -149,9 +147,9 @@ const create = (customConfig = {}) => {
 
       if (Utils.isHotMode(c)) F.mapObj(entries, Utils.addChunkDevServer)
 
-      /* Generate the Jade template. */
+      /* Generate the Jade rendering function. */
 
-      const template = createTemplate(config.templateExtension)
+      const render = createRenderer(config.jade, config.templateExtension)
```

Comment:
Ahh, thanks — I wasn't sure if you were thinking that would be reserved for the Webpack compiler or the Jade compiler. Updated!